.travis.yml

```diff
@@ -71,3 +71,24 @@ matrix:
       after_failure:
         - adb logcat -d > logcat.txt
         - echo "$(cat logcat.txt)"
+
+    - language: rust
+      sudo: false
+      cache:
+        - cargo
+      rust:
+        - stable
+      before_script:
+        - (test -x $HOME/.cargo/bin/cargo-install-update || cargo install cargo-update)
+        - (test -x $HOME/.cargo/bin/mdbook || cargo install --vers "^0.3" mdbook)
+        - cargo install-update -a
+      script:
+        - mdbook build ./docs
```

Comment:
one more thought: perhaps it makes sense to move those script items to `script/` folder? So that outside of CI one can easily run this?

Reply:
added a bash script in ./scripts so you can run `yarn run docs`